### PR TITLE
Add SystemML to OSS Fuzz

### PR DESCRIPTION
**Pull request by j143**
1. SystemML contains a part of it written in cpp, while most other is in Java. There are plans to add cpp backend, end of this year.
2. This program is counterpart to tensorflow.

### CODE REVIEW

**Comment by inferno-chromium** [comment]
@mboehm7 - are you ok with this ? Should your name be added to cc list as well ?

**Comment by j143** [comment]
@inferno-chromium once oss-fuzz gives green signal, I will the results will only be cc'ed to our private mailing list.

Ping @niketanpansare , @nakul02

File: projects/systemml/project.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+homepage: "https://github.com/apache/systemml"
+primary_contact: "janardhan@apache.org"
+auto_ccs:
+  - "janardhan@apache.org"
```
Comment (inferno-chromium) [inline projects/systemml/project.yaml]: No need to repeat same email in auto-cc. Add the mailing list as you cited in comment.

**Comment by j143** [comment]
Hi @inferno-chromium , thanks for green signal. After approval from our community the results will only be cc'ed to our private mailing list ( private@systemml.apache.org ), according Apache guidelines, not to an individual mail address.

Please give us a little time, to allow for everyone's opinion.
Thanks a lot. :)

**Comment by kcc** [comment]
You will still require an individual email to be listed in project.yaml in order to login into oss-fuzz.com. 
Email notifications don't contain all the information that you can see at oss-fuzz.com, and 
oss-fuzz.com can not authenticate a mailing list.

**Comment by mboehm7** [comment]
Thanks for asking @inferno-chromium. Before we go any further, we need to do our homework and define a set of meaningful fuzz targets first. Right now, SystemML handles all external inputs and I/O in Java and only calls out for certain operations to native CPU and GPU operations. Although I see use cases for fuzz testing there (e.g., corrupted column indexes in sparse inputs), I don't think it's a good idea to perform additional checks on each operation instead of hardening the input paths into the system. We'll come back in the next days.

**Comment by inferno-chromium** [comment]
Thanks @j143 and @mboehm7

**Comment by ryangueraa** [comment]
Nice

**Comment by Dor1s** [comment]
@mboehm7 friendly ping.

**Comment by j143** [comment]
Hi, we just prepared a Docker image. And are still in the process of identifying fuzz targets for our java program. 

@Dor1s - Max, can we merge only `projects.yaml` file for now. Can you guide us, with targets in our [project](http://github.com/apache/systemml).

**Comment by kcc** [comment]
you may find  https://da-data.blogspot.com/2017/01/finding-bugs-in-tensorflow-with.html useful

**Comment by j143** [comment]
Hi oss-fuzz team, thank you all for you help and tips. After going through understanding the fuzzing, and and taking tutorials suggested by you and tweaking with some of the fuzz projects here (systemd, zlib, .) -- I am finding myself technically incompetent to handle this task for now. Once, I could handle I will try to do my best with this PR.

For now, can this be **closed**. _Very sorry_ for inconvenience.

**Pull request closed without merging**